src/CommandRunner.cpp

```diff
@@ -0,0 +1,204 @@
+#include "CommandRunner.hpp"
+#include <memory>
+#include <stdio.h>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include <signal.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <time.h>
+
+#include <glog/logging.h>
+#include <stout/try.hpp>
+
+#define TEMP_FILE_TEMPLATE "/tmp/criteo-mesos-XXXXXX"
```

Comment:
FAR: I think we should use a better location that bare /tmp.

Are you sure that this directory is cleaned automatically (in case your program fail to do so)?

Configuration on centos7 seems to be 
```
v /tmp 1777 root root 10d
v /var/tmp 1777 root root 30d
```
(for systemd builtin cleaning tmp files)